README.md

```diff
@@ -56,10 +56,12 @@ and [contributing guidelines](CONTRIBUTING.md).
 
 ## Quickstart
 
-Run this command to start FerretDB with PostgreSQL:
+Run this command to start FerretDB with PostgreSQL, ensure to update `<username>` and `<password>`:
 
 ```sh
 docker run -d --rm --name ferretdb -p 27017:27017 --platform linux/amd64 \
+  --env POSTGRES_USER=<username> \
+  --env POSTGRES_PASSWORD=<password> \
```

Comment:
We generally use short flags when they are available; for example, `-d` instead of `--detach`. Let's use `-e`.